cocos/2d/CCRenderTexture.cpp

```diff
@@ -252,17 +259,62 @@ bool RenderTexture::initWithWidthAndHeight(int w, int h, Texture2D::PixelFormat
 
         if (depthStencilFormat != 0)
         {
+            
+                        
+#if(CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID)
+            if(Configuration::getInstance()->supportsOESPackedDepthStencil())
+            {
+                //create and attach depth buffer
+                glGenRenderbuffers(1, &_depthRenderBufffer);
+                glBindRenderbuffer(GL_RENDERBUFFER, _depthRenderBufffer);
+                glRenderbufferStorage(GL_RENDERBUFFER, depthStencilFormat, (GLsizei)powW, (GLsizei)powH);
+                glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_RENDERBUFFER, _depthRenderBufffer);
+
+                // if depth format is the one with stencil part, bind same render buffer as stencil attachment
+                if (depthStencilFormat == GL_DEPTH24_STENCIL8)
+                {
+                    glFramebufferRenderbuffer(GL_FRAMEBUFFER, GL_STENCIL_ATTACHMENT, GL_RENDERBUFFER, _depthRenderBufffer);
+                }
+            }
+            else
+            {
+
+                glGenRenderbuffers(1, &_depthRenderBufffer);
+                glGenRenderbuffers(1, &_stencilRenderBufffer);
+                glBindRenderbuffer(GL_RENDERBUFFER, _depthRenderBufffer);
+                
+                if(Configuration::getInstance()->supportsOESPackedDepthStencil())
```

Comment:
need to check whether an android device supports both supportsOESDepth24 and supportsOESPackedDepthStencil :)